Javascript-2/Week-1/Movie App/index.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Movie Streaming Application</title>
+    <link rel="stylesheet" , href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.2/css/all.min.css"
+      integrity="sha512-SnH5WK+bZxgPHs44uWIX+LLJAJ9/2PkPKZ5QiAj6Ta86w+fsb2TkcmfRyVX3pBnMFcV7oQPJkl9QevSCWr3W6A=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <script type="module" src="script.js" defer></script>
+    <script src="arrayList.js"></script>
+</head>
+<body>
+    <header>
+        <!-- <img
+          src="https://cdn2.iconfinder.com/data/icons/video-36/100/video-film-roll-video-player-media-app-film-roll-movie-512.png"
+          ,
+          width="100"
+          ,
+          height="100"
+        /> -->
+
+        <p class="site-name"><i class="fa fa-video-camera col_red me-1" style="margin-right: 10px"></i>Flimy Fly</p>
+  
+        <!-- <nav class="nav">
+          <ul>
+            <li><a href="">Home</a></li>
+            <li><a href="">Movies</a></li>
+            <li><a href="">Series</a></li>
+          </ul>
+        </nav> -->
+        <div class="forUser">
+          <!-- <span>
+            <i id="shorting" , class="fa-solid fa-list"></i>
+            <fieldset id="filters">
+              <p id="newRelease" class="filter">New Releases</p>
+              <p id="actors" class="filter">Actors</p>
+              <p id="releaseYear" class="filter">Release Year</p>
+              <label
+                ><select id="dropdown">
+                  <option>select one</option>
+                  <option>1970-1979</option>
+                  <option>1980-1989</option>
+                  <option>1990-1999</option>
+                  <option>2000-2009</option>
+                  <option>2010-2019</option>
+                  <option>2020-Till Now</option>
+                </select></label
+              >
+              <p id="genres" ,class="filter">genres</p>
+            </fieldset>
+          </span> -->
+          <!-- <span> <i class="fa-solid fa-bell"></i></span> -->
+          <span>
+            <input
+              type="search"
+              ,
```

Comment:
You don't need these commas between each attribute of an HTML element. I am commenting on it here, but I have seen them a bit everywhere.